### DISCUSSION TITLE
OIDCStateCookiePrefix config option no longer recognized after upgrade to 2.4.16

### DISCUSSION THREAD

**Discussion by damisanet** [OpenIDC/mod_auth_openidc#1254] [Q&A] [answered]
Hi,

it seems that somewhere between 2.4.15.7 and 2.4.16 `OIDCStateCookiePrefix` stopped being recognized:

```
Invalid command 'OIDCStateCookiePrefix', perhaps misspelled or defined by a module not included in the server configuration
```

I guess this may be a bug related to refactor of config handling in https://github.com/OpenIDC/mod_auth_openidc/commit/f32d12ebb159e7171967501d85fce17b1db9f8df, as this option is still mentioned in `auth_openidc.conf` and there are calls to `oidc_cfg_dir_state_cookie_prefix_get` in code.

**Comment by zandbelt** [comment, marked as answer]
that is a bug due to the rewrite indeed, fixed just now in 47ece7fa9ee289e53e8c1330731971fe1cacb5e8 ; it will be released as part of 2.4.16.2 soon, thanks
